src/lib/authserver/ui/new_dir_inst.rb

```diff
@@ -129,7 +133,7 @@ def ok_handler
       UI.ReplaceWidget(Id(:busy), Label(_('Configuring instance TLS ...')))
       # Turn on TLS
       if !DS389.install_tls_in_nss(instance_name, tls_ca, tls_p12)
-        Popup.Error(_('Failed to set up new instance! Log output may be found in /var/log/YaST/y2log'))
+        Popup.Error(_('Failed to set up new instance! Log output may be found in %{y2log}'), y2log: y2log)
```

Comment:
BTW, you are always reporting the same error here. So, extracting it to a method makes more sense to me.

```ruby
def report_error
  Popup.Error(
    format(
      # TRANSLATORS: %{y2log_path} will be replaced for the YaST2 log path. Please, keep it in the translation, curly braces included.
      _('Failed to set up new instance! Log output may be found in %{y2log_path}'),
      y2log_path: '/var/log/YaST2/y2log'
    )
  )
end
```

And then just using where needed

```ruby
if !DS389.install_tls_in_nss(instance_name, tls_ca, tls_p12)
  report_error
  ...
```

Thus, you can drop the `y2log` method since you are using the right path in the proposed `report_error`. To know more about the `#TRANSLATORS` comment, please read https://yastgithubio.readthedocs.io/en/latest/localization/#notes-for-translators

But, of course, this is just a suggestion, not needed to do so for getting my approval for this PR.